crates/ruma-client-api/src/r0/session/sso_login_with_provider.rs

```diff
@@ -9,7 +7,7 @@ ruma_api! {
         description: "Get the SSO login identity provider url.",
         method: GET,
         name: "sso_login_with_provider",
-        path: "/_matrix/client/unstable/org.matrix.msc2858/login/sso/redirect/:idp_id",
+        path: "/_matrix/client/v3/login/sso/redirect/:idp_id",
```

Comment:
It probably makes sense to use this path since this is a new route, but then the doc comment at the top of the file should also be updated, and I think for clarity this module should already be moved out of `r0::...` to `session::sso_Login_with_provider::v3`.

Reply:
Sure.